packages/rich-text/src/test/register-format-type.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ *  Internal dependencies
+ */
+import { registerFormatType } from '../register-format-type';
+import { unregisterFormatType } from '../unregister-format-type';
+import { getFormatType } from '../get-format-type';
+import { getFormatTypes } from '../get-format-types';
+
+describe( 'registerFormatType', () => {
+	const defaultFormatSettings = { edit: noop, title: 'format title' };
+
+	// Initialize format store.
+	require( '../store' );
```

Comment:
`beforeAll` again :)
I also noticed that comment isn't 100% accurate. It is rich text store.

In general, we need to come up with a nicer way of using stores with tests.